docs/src/tutorials/shapley.md

```diff
@@ -100,6 +100,8 @@ copula = GaussianCopula(Covmat)
 input_distribution = SklarDist(copula, marginals)
 
 function batched_loss_n_ode(θ)
+    # The copula returns samples of `Float64`s
+    θ = convert(AbstractArray{Float32}, θ)  
```

Comment:
I think this is an upstream problem - the `copula` (and hence also `input_distribution`) lost the type information and generate samples of type `Float64`.

Reply:
yup makes sense cc:@lrnv

Reply:
Yep this is a current bug. Distributions.jl does'nt always get it right too, see e.g. [there](https://github.com/JuliaStats/Distributions.jl/issues/1783).

Opened [an issue](https://github.com/lrnv/Copulas.jl/issues/195), will look at it when i have a bit of time (not right now, sorrry).